examples_tests/48.ArithmeticUnitTest/common.glsl

```diff
@@ -0,0 +1,35 @@
+
+#include "irr/builtin/glsl/subgroup/arithmetic_portability.glsl"
+
```

Comment:
you're missing `layout (workgroup_size_x=_IRR_GLSL_WORKGROUP_SIZE_) in;` to actually declare the size of the workgroup to GLSL

Reply:
`workgroup_size_x` didnt exist, so i used `local_size_x `